test/unit/foreman/renderer_test.rb

```diff
@@ -1,475 +0,0 @@
-  end
-
-  context 'subnet helpers' do
-    setup do
-      @renderer.host = FactoryBot.build(:host, :with_puppet)
-      subnets(:one).subnet_parameters.create(name: 'myparam', value: 'myvalue')
-    end
-
-    test 'should have subnet_has_param? helper returning true' do
-      assert @renderer.subnet_has_param?(subnets(:one), 'myparam')
-    end
-
-    test 'should have subnet_has_param? helper returning false' do
-      refute @renderer.subnet_has_param?(subnets(:one), 'my_wrong_param')
-    end
-
-    test 'should have subnet_has_param? helper returning false when subnet is nil' do
-      assert_raises Foreman::Renderer::WrongSubnetError do
-        @renderer.subnet_has_param?(nil, 'myparam')
-      end
-    end
-
-    test 'should render existing subnet param using "subnet_param" helper' do
-      assert_equal @renderer.subnet_param(subnets(:one), 'myparam'), 'myvalue'
-    end
-
-    test 'should not render missing subnet param using "subnet_param" helper' do
-      assert_nil @renderer.subnet_param(subnets(:one), 'my_wrong_param')
-    end
-
-    test 'should throw an error using "subnet_param" helper with nil' do
-      assert_raises Foreman::Renderer::WrongSubnetError do
-        @renderer.subnet_param(nil, 'myparam')
-      end
-    end
-  end
-
-  test 'should have host_enc helper' do
-    @renderer.host = FactoryBot.build(:host, :with_puppet)
-    assert @renderer.host_enc
-  end
-
-  test "should find path in host_enc" do
-    host = FactoryBot.build(:host, :with_puppet)
-    @renderer.host = host
-    assert_equal host.puppetmaster, @renderer.host_enc('parameters', 'puppetmaster')
-  end
-
-  test "should raise rendering exception if no such parameter exists while rendering host_enc" do
-    host = FactoryBot.build(:host, :with_puppet)
-    @renderer.host = host
-    assert_raises(Foreman::Renderer::HostENCParamUndefined) do
-      assert_equal host.puppetmaster, @renderer.host_enc('parameters', 'puppetmaster_that_does_not_exist')
-    end
-  end
-
-  test 'should raise rendering exception if @host is not set while rendering host_enc' do
-    @renderer.host = nil
-    assert_raises(Foreman::Renderer::HostUnknown) do
-      @renderer.host_enc('parameters', 'puppetmaster')
-    end
-  end
-
-  test 'templates_used is allowed to render for host' do
-    assert Safemode.find_jail_class(Host::Managed).allowed? :templates_used
-  end
-
-  test "global_setting unsafe attempt" do
-    assert_raises(Foreman::Renderer::FilteredGlobalSettingAccessed) do
-      setup_safemode_renderer
-      @renderer.render_safe('<%= global_setting("not_allowed_setting") %>', [:global_setting])
-    end
-  end
-
-  private
-
-  def assert_renders(template_content, output, host)
-    @renderer.host = host
-    template = mock('template')
-    template.stubs(:template).returns(template_content)
-    assert_nothing_raised do
-      content = @renderer.unattended_render(template)
-      assert_equal(output, content)
-    end
-  end
-end
```

Comment:
Looks like 475 lines of tests were removed but 506 lines added. Numbers look good, did you intentionally drop any test from this file?

Reply:
I just moved all the tests to the new structure, I think I didn't miss any of them.